reconcile/ocm_upgrade_scheduler.py

```diff
@@ -176,6 +177,23 @@ def version_conditions_met(version, history, ocm_name, workloads, upgrade_condit
     return conditions_met
 
 
+def upgradeable_version(policy: dict, history: dict, ocm: OCM) -> Optional[str]:
```

Comment:
:heart: that this was extracted to an extra method.

small nit: According to style guide we should favor abstract collection types in func args

```
def upgradeable_version(policy: Mapping, history: Mapping, ocm: OCM) -> Optional[str]:
```

Reply:
thanks, fixed in 42b01ae